include/bluetooth/bluetooth.h

```diff
@@ -130,10 +130,13 @@ typedef void (*bt_ready_cb_t)(int err);
  * Enable Bluetooth. Must be the called before any calls that
  * require communication with the local Bluetooth hardware.
  *
- * When @kconfig{CONFIG_BT_SETTINGS} has been enabled and the application is not
- * managing identities of the stack itself then the application must call
- * @ref settings_load() before the stack is fully enabled.
- * See @ref bt_id_create() for more information.
+ * When @kconfig{CONFIG_BT_SETTINGS} is enabled, the application must load the
+ * Bluetooth settings after this API call successfully completes before
+ * Bluetooth APIs can be used. Loading the settings before calling this function
+ * is insufficient. Bluetooth settings can be loaded with settings_load() or
```

Comment:
Maybe we could even explicitly state that Bluetooth settings will be skipped if settings_load() is called before Bluetooth is enabled.